crates/pera-json-rpc-types/src/pera_transaction.rs

```diff
@@ -467,6 +468,7 @@ impl Display for PeraTransactionBlockKind {
             Self::DWalletMPCOutput(_) => {
                 writeln!(writer, "Transaction Kind: dwallet mpc Output")?;
             }
+            _ => {}
```

Comment:
Don't use `_` as it may cause potential debugging difficulties in the future.

```suggestion
           PeraTransactionBlockKind::LockNextCommittee => {
                writeln!(writer, "Transaction Kind: Lock Next Committee")?;
            }
```